### ISSUE TITLE
Marvelloous Suspender dosen't work after Chrome updates in August 2022

### ISSUE THREAD

**Issue by SteveL9806** [deanoemcke/thegreatsuspender#1356]
Please complete the following information when submitting a feature request or bug report.
 * Extension version:  Marvellous Suspender v 7.1.5.2
 * Browser name & version:  Chrome 104.0.5112.80
 * Operating system & version:  Windows 10 Pro,  V 21H2,  OS 19044.1826

I had 6 windows with 40 - 60 tabs, all suspended by Marvellous Suspender when Chrome upgraded itself in August 2022. Lost all tabs but found a way to re-establish them.  This worked until I rebooted, when the problem re-occured.  This problem is apparently know to many members here.  I'm just requesting on getting this fixed permanently.  I'm just a computer used, not a developer or code writer, just looking for help.

**Comment by markwyner** [comment]
I can confirm that I have experienced the same issue. My details for troubleshooting:

- Extension version: 7.1.6.2
- Browser: Chrome 104.0.5112.79 (Official Build) (x86_64)
- OS: macOS Monterey 12.5 (21G72)

My experience:

- All of the tab previews show the suspender icon instead of the respective favicon
- When any tab is given focus the screen is blank (usually showing suspender details)
- Clicking the white space in the tab two times brings back the suspended tab
- After bringing all tabs back to life, things work as expected until the browser or computer are restarted, at which time this process starts over again

**Comment by efranqueiro** [comment]
Same here.
Estension version: 7.1.6
Browser: Chrome 104.0.5112.79 (Official Build) unknown (64-bit)
OS: Fedora 36

**Comment by wjcroft** [comment]
Seeing the same here with macOS and Chrome.
Extension version: 7.1.6.2, April 2 2021
Chrome: Version 104.0.5112.79 (Official Build) (x86_64)
macOS: Catalina 10.15.7
machine: Mac Mini, late 2012

**Comment by chrisi51** [comment]
This might be the release notes but im not capable to see there any problems.
https://chromereleases.googleblog.com/2022/08/stable-channel-update-for-desktop.html

I also don't think that any developers still have this project on their list. =(
Btw. there are multiple clones based on this repo but all of them seem to have the same problem.

**Comment by pa-jberanek** [comment]
https://github.com/gioxx/MarvellousSuspender/issues/185

**Comment by pa-jberanek** [comment]
tl;dr of above...it's fixed in a later Chrome release.
